app/layout.tsx

```diff
@@ -10,8 +10,8 @@ export default function RootLayout({
 }) {
   return (
     <html lang="en">
-      <Navbar />
       <body>{children}</body>
+      <Navbar />
```

Comment:
Just confirming here, guessing your intention is to have a layout where Navbar is sticky to the bottom of the page?

Reply:
Switched Navbar component position